docs/get-started/index.md

```diff
@@ -167,6 +162,10 @@ flux create source git webapp \
   --export > ./staging-cluster/webapp-source.yaml
 ```
 
+## 3. Configure a kustomization for the applications
```

Comment:
If we want to simplify this then it should be a single Kustomization that reconciles podinfo in the default namespace. Something like:

```
flux create source git webapp \
  --url=https://github.com/stefanprodan/podinfo \
  --branch=master \
  --interval=30s \
  --export > ./clusters/staging/podinfo.yaml
```

And

```
flux create kustomization webapp-common \
  --source=webapp \
  --path="./kustomize" \
  --prune=true \
  --validation=client \
  --interval=5m
```

Reply:
I think this is a good idea

Reply:
What about the rest? flux get kustomization and everything else is still showing the old webapp, not to mention the paths are broken, the bootstrap should be for `clusters/staging`, the kustomization is not written to git...

Reply:
Still updating the docs atm

Reply:
> The bootstrap should be for cluster/staging

I was going to rename the section to a multi-cluster setup and add both staging and production.
You want me to limit it to just staging?

Reply:
> Still updating the docs atm

Ah ok, you marked this conversation  as resolved, so I got confused.

PS. Bootstrap should be run for `./clusters/staging` so that it uses the same repo structure as the [installation instructions](https://toolkit.fluxcd.io/guides/installation/#bootstrap) and the 2 example repos.